src/service/QuestionGenerator.js

```diff
@@ -0,0 +1,50 @@
+import { randomNumberInRange } from './randomNumberInRange';
+import {QuestionService} from './QuestionService';
+import { WHO_IS_THAT_POKEMON } from "./modes";
+
+const questionService = new QuestionService()
+
+export class QuestionGenerator {
+
+    constructor(mode) {
+        this.mode = mode;
+        this.minPokeId = 1;
+        this.maxPokeId = 152;
+        this.askedQuestions = [];
+        this.askedQuestionsCount = 0;
+    };
+
+    async getNextQuestion() {
+        if (this.askedQuestionsCount === 30) {
+            return undefined;
+        }
+        this.askedQuestionsCount++;
+        console.log("this.askedQuestionsCount: " + this.askedQuestionsCount)
+
+        const correctAnswerIndex = randomNumberInRange(0, 3);
+        questionService.correctAnswerIndex = correctAnswerIndex;
```

Comment:
this is part of randomising inside of QuestionService (and there is also a task for that). Do not touch insides of questionService on the outside :)

Reply:
I removed this part of code. As a result getNextQuestion always returns object where question '0' is proper. The same like getQuestion method.